No need for the reader to have GitHub account

andynava opened the pull request:
The way the quickstart is worded, it does not require the reader to use their own GitHub account, so I think this requirement should be removed from the documentation.

Another option would be to add wording that references the reader's GitHub account...

The _deployment-source-url_ parameter is a sample repository in GitHub that contains a "Hello World" HTTP triggered function. You can replace this URL with your own GitHub repository that contains your function code.

azurecla commented:
Hi __@jeffhollan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

andynava committed:
No need for the reader to have GitHub account

The way the quickstart is worded, it does not require the reader to use their own GitHub account, so I think this requirement should be removed from the documentation.

Another option would be to add wording that references the reader's GitHub account...

The _deployment-source-url_ parameter is a sample repository in GitHub that contains a "Hello World" HTTP triggered function. You can replace this URL with your own GitHub repository that contains your function code.
